variables.tf

```
@@ -62,6 +62,12 @@ variable "service_account_role_arn_annotation_enabled" {
   default     = true
 }
 
+variable "service_account_set_key_path" {
+  type        = string
+  description = "Key path used in `helm --set` to point to the service account in the chart values (ex.: `serviceAccount` or `rbac.serviceAccount`)"
```

I feel like this was a little too technical (i.e. `helm --set`) and this will be more concise:

```suggestion
  description = <<-EOT
  The key path used by Helm Chart values for ServiceAccount-related settings (e.g. `serviceAccount` or `rbac.serviceAccount`).
  
  Ignored if either `var.service_account_role_arn_annotation_enabled` or `var.iam_role_enabled` are set to `false`.
  EOT
```